packages/styles/settings-tools/_t.state-classes.scss

```diff
@@ -0,0 +1,31 @@
+@mixin manage-state-classes {
```

Comment:
maybe `add-demo-state-classe` define more clearly the intent.

Reply:
Fixed